lib/normalize.js

```diff
@@ -32,6 +33,16 @@ var _splitLocator = function (nemo, locatorString) {
 module.exports = function normalize(nemo, _locator) {
   var locator = _locator;
   var normalizedLocator;
+
+  if (locator.using && locator.value && nemo.wd.By[locator.using]) {
+    normalizedLocator = nemo.wd.By[locator.using](locator.value);
+    if(normalizedLocator.constructor === locator.constructor){
```

Comment:
I'm not sure I understand why this conditional is even necessary. if `normalizedLocator` and `locator` are `===` you can just return `normalizedLocator` regardless of whether the condition is true or false.